ScnSoft-Education/LeetCodeTraining.Tests/ReverseIntegerSolutionTests.cs

```diff
@@ -0,0 +1,20 @@
+using LeetCodeTraining.ReverseInteger;
+
+namespace LeetCodeTraining.Tests
+{
+    public class ReverseIntegerSolutionTests
+    {
+        [Theory]
+        [InlineData(123,321)]
```

Comment:
apply "format document" always before commit your code
In Visual studio -> Edit -> Advanced -> format document
or use hot key ctrl + k, d

Reply:
"Format document" was applied